docs/reference/commandline/login.md

```diff
@@ -144,9 +144,9 @@ For example:
 ```json
 {
   "credHelpers": {
-    "registry.com": "registryhelper",
-    "kontainr.ly": "hip-star",
-    "sfo.unicorn.com": "vcbait"
+    "example.registry.com": "registryhelper",
+    "example.awesomereg.io": "hip-star",
```

Comment:
oh, I meant to use `example.com` or `example.org` as domain name. I know it _should_ be clear to users, but we've had situations where people used the literal examples, and the first domain is actually active (second one is available to buy if you're interested :smile:);

```
Domain Name: REGISTRY.COM
Registrar: GODADDY.COM, LLC
...
Expiration Date: 18-nov-2018
```


perhaps `awesomereg.example.org` ?